app/views/maths_and_physics/claims/_ineligibility_reason_no_entire_term_contract.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1 class="govuk-heading-xl">
+  You’re not eligible for this payment
+</h1>
+
+<p class="govuk-body">
+  You can only get this payment if you are employed directly by your school for
+  at least 1 term.
```

Comment:
The [GOV.UK Style Guide](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#numbers) seems to suggest we use 'one' in this case.

Reply:
Yeah, it rankled a bit but I didn't want to change it just because of personal preference. Good to know about the Style Guide, thanks! Will fix.